### PR TITLE
Client Meeting Notes for W9

### PR DESCRIPTION
**Pull request by thenewguy321**
This PR uploads all the Client Meeting Notes for W9, detailing any discussions/questions that the client might have for our group.

### CODE REVIEW

**Comment by thenewguy321** [comment]
Close #14

File: documentation/meetings/clientMeetings/2022 11 01 - Client Meeting W9 [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# 2022/11/01 - Client Meeting W9
+
+Created by: Anonymous
+
+- This image is a showcase from the other teams that they showed to Matthias
```
Comment (Brenden-Trieu) [inline documentation/meetings/clientMeetings/2022 11 01 - Client Meeting W9:5]: what is the image in question?

File: documentation/meetings/clientMeetings/2022 11 01 - Client Meeting W9 [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    - Will the user be able to see the video being uploaded?
+    - This is actually the images from the video, shown on the RHS
```
Comment (Brenden-Trieu) [inline documentation/meetings/clientMeetings/2022 11 01 - Client Meeting W9:7]: RHS?

File: documentation/meetings/clientMeetings/2022 11 01 - Client Meeting W9 (hunk 1 above)
Comment (thenewguy321) [reply documentation/meetings/clientMeetings/2022 11 01 - Client Meeting W9:5]: Discussed and closed

File: documentation/meetings/clientMeetings/2022 11 01 - Client Meeting W9 (hunk 2 above)
Comment (thenewguy321) [reply documentation/meetings/clientMeetings/2022 11 01 - Client Meeting W9:7]: Right Hand Side

**Review by Brenden-Trieu — APPROVED** [review]
LGTM

**Review by yuqi88 — APPROVED** [review]
LGTM!

**Comment by yuqi88** [comment]
Thanks for covering for me this week

**Pull request merged**